ivy/functional/frontends/jax/func_wrapper.py

```diff
@@ -1,11 +1,12 @@
 # global
 import functools
 from typing import Callable
+import inspect
 
 # local
 import ivy
 import ivy.functional.frontends.jax as jax_frontend
-
+import ivy.functional.frontends.numpy as np_frontend
 
 def _is_jax_frontend_array(x):
```

Comment:
small lint fix, add a line above this function

Reply:
Will do cleanup at the end including lints etc. 😊